sdk/contentsafety/ai-content-safety-rest/package.json

```diff
@@ -42,7 +42,7 @@
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"*.{js,json}\"  \"test/**/*.ts\"",
     "generate:client": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "integration-test:browser": "npm run clean && dev-tool run build-package && dev-tool run build-test && dev-tool run test:vitest --browser",
+    "integration-test:browser": "echo skipped",
```

Comment:
nit: should we remove `vitest.browser.config.ts` since tests only support NodeJS?

Reply:
Fixed.